.github/workflows/sync-flow-go.yml

```diff
@@ -70,11 +83,11 @@ jobs:
               Auto generated PR to update Cadence version.
 
               References: ${{ github.event.pull_request.html_url }}
-          branch: auto-cadence-upgrade/${{ github.event.pull_request.head.ref }}
+          branch: ${{ env.NEW_BRANCH }}
           delete-branch: true
 
           # we can tweak the following:
           # labels: A comma or newline-separated list of labels.
           # assignees: A comma or newline-separated list of assignees (GitHub usernames).
           # reviewers: A comma or newline-separated list of reviewers (GitHub usernames) to request a review from.
-          assignees: janezpodhostnik
\ No newline at end of file
+          assignees: janezpodhostnik,ramtinms,turbolent
```

Comment:
Nice! As I am the creator of the PR, I am not able to approve "my own" PRs, but adding Ramtin is a good idea. Maybe also add @SupunS? Most of the time, the PRs are just simple version bumps that can be easily approved without any knowledge about flow-go/FVM